analyzer-core/src/main/java/ffb/analyzer/core/pubsub/Subscriber.java

```diff
@@ -0,0 +1,7 @@
+package ffb.analyzer.core.pubsub;
+
+import java.util.Map;
+
+public interface Subscriber {
+    public void update(String publishedResult);
```

Comment:
how about `boolean` as a result? Then the subscriber can format that however it wants?

Reply:
I did string so we could return JSON strings. Are we going to store every JSON value we retrieve into the database / a file? If not, we'd need to pass the result itself to use the info. I assumed there would be certain temporary info we wouldn't want to store.